tests/test_atom_mapping.py

```diff
@@ -988,3 +989,22 @@ def test_min_threshold():
 
     with pytest.raises(NoMappingError, match="Unable to find mapping with at least 18 atoms"):
         atom_mapping.get_cores(mol_a, mol_b, **core_kwargs, max_visits=10000)
+
+
+def test_get_cores_and_diagnostics():
+    mols = Chem.SDMolSupplier(hif2a_set, removeHs=False)
```

Comment:
nit: use `read_sdf`

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1231/commits/5950ca667493bba1e737e9d51396eda35d1b4224 (was just copied; fixed in both locations)